queries/unison/highlights.scm

```diff
@@ -0,0 +1,75 @@
+;; Primitives
+(comment) @comment
+(nat) @number
+(unit) @constant.builtin
+(literal_char) @character
+(literal_text) @string
+(literal_boolean) @boolean
+
+;; Keywords
+[
+  (kw_forall)
+  (unique_kw)
+  (type_kw)
+  (kw_equals)
+  (do)
+] @keyword
+
+(kw_let) @keyword.function
+(structural_kw) @storageclass
+(unique) @storageclass
+
+[ 
+  (type_constructor)
+] @constructor
+
+(doc_block) @comment
+
+
+[
+  (operator)
+  (pipe) 
+  (arrow_symbol) 
+  (">")
+  (or)
+] @operator
+
+[
+  "if"
+  "else"
+  "then"
+  (match)
+  (with)
+  (cases)
+] @conditional
+
+(blank_pattern) @variable.builtin
+
+((pattern) @variable (constructor_or_variable_pattern) @type)
```

Comment:
not sure what the wrapped parenthesis indicates, there's no parent node to these two

Reply:
Can these kind of patterns deduct consecutive nodes, or will they always be resolved to 2 separate query captures

Reply:
Removed the outer parens from this. Also tried the outer parens on the next line `(record_field name: (wordy_id) @variable type: (wordy_id) @type)` but that caused things to break.

Reply:
Ah I see, my bad. Were you intending to match two items at the root level? Using the first/root rule as the parent would be much clearer then if so, otherwise if you meant for this to occur anywhere it'd be interesting to know the use-case/where it's used, I've never seen that kind of construct not wrapped in a parent rule of sorts except for immediate following matches w/ anchors (like several comments together)

Reply:
> Can these kind of patterns deduct consecutive nodes, or will they always be resolved to 2 separate query captures

Consecutive nodes allowing anything in-between as long as they're at the same level (top level, inside a definition of sorts, etc)